sphinx/source/docs/user_guide/export.rst

```diff
@@ -68,35 +65,33 @@ saving the image to a file, you can use the lower-level function
 
 .. _userguide_export_svg:
 
-SVG Generation
+SVG generation
 --------------
 
-Bokeh also supports replacing the HTML5 Canvas plot output with an SVG element
-that can be edited in image editing programs such as Adobe Illustrator and/or
-converted to PDFs.
+Bokeh can also replace HTML5 Canvas plot output with an Scalable Vector
+Graphics (SVG) element that can be edited in image editing programs such
+as Adobe Illustrator and/or converted to a PDF.
 
 The SVG output isn't as performant as the default Canvas backend when it comes
 to rendering a large number of glyphs or handling lots of user interactions like
 panning.
 
-Like PNGs, in order to create an SVG with a transparent background, users
-should set the ``Plot.background_fill_color`` and ``Plot.border_fill_color``
-properties to ``None``.
+To create an SVG with a transparent background, set the ``Plot.background_fill_color``
+and ``Plot.border_fill_color`` properties to ``None``, same as with PNG output.
 
 Limitations
 ~~~~~~~~~~~
 
-It's not possible to create a single SVG for a layout of plots, as each plot
-is its own distinct SVG element. Alternatively, it's possible to download an
-SVG plot using a SaveTool from the toolbar. However, currently an SVG export
-of a plot with a toolbar will have a blank area where the toolbar was rendered
-in the browser.
+You can't create a single SVG for a layout of plots because each plot will
+produce its own distinct SVG element. You can, however, download an SVG plot
+using a SaveTool from the toolbar although an SVG export of a plot with a
```

Comment:
very long sentence with lots of modifiers. Maye somthing like: "... using a SaveTool from the toolbar. If you use this option to create an SVG export and your plot contains a toolbar, keep in mind that the exported file will have a blank area ..."

Reply:
Okay, yeah. This one is a bit heavy-handed. Broke it up and simplified.